conjur/cli.py

```diff
@@ -79,52 +92,70 @@ def run(self, *args):
         resource_subparsers = parser.add_subparsers(dest='resource', title=self.title("Commands"))
 
         resource_subparsers.add_parser('whoami',
-            help='Provides information about the current logged-in user')
+        help='Provides information about the current logged-in user')
 
         resource_subparsers.add_parser('list',
-            help='List all available resources belonging to this account')
+        help='List all available resources belonging to this account')
 
         variable_parser = resource_subparsers.add_parser('variable',
-            help='Manage variables')
+                                                         help='Manage variables',
```

Comment:
We have two options here:
1. You take my recently merged PR as an example for how to build your help screens (git fetch, git rebase origin/master
2. Ignore the help screens and I will take care of it later

I will support whichever you choose 😄